File: c/PRAK304-2310817310004-MSAMILRENDYNORSALEH.c
#include <stdio.h>

int main() {
    int N;

    printf("");
    scanf("%d", &N);

    if (1 <= N && N <= 9) {
        printf("Satuan\n");
    } else if (11 <= N && N <= 19) {
        printf("Belasan\n");
    } else if (10 <= N && N <= 99) {
        printf("Puluhan\n");
    } else if (N >= 100) {
        printf("Anda Menginput Melebihi Limit Bilangan\n");
    } else {
        printf("Nol\n");
    }

    return 0;
}

File: python/PRAK304-2310817310004-MSAMILRENDYNORSALEH.py
N = int(input(""))

if 1 <= N <= 9:
    print("Satuan")
elif 11 <= N <= 19:
    print("Belasan")
elif 10 <= N <= 99:
    print("Puluhan")
elif  N >= 100:
    print("Anda Menginput Melebihi Limit Bilangan")
else:
    print("Nol")